test/directives/vector_directive1.f90

```diff
@@ -1,4 +1,4 @@
-! RUN: %flang -c %s 2>&1 | FileCheck %s --check-prefix=CHECK-NO-CLAUSE
+! RUN: %flang -S -emit-llvm %s 2>&1 -o - | FileCheck %s --check-prefix=CHECK-NO-CLAUSE
```

Comment:
Is this change necessary?

Reply:
The `-o -` part is necessary. Without that, the test fails with the following:
```
FileCheck error: '<stdin>' is empty.
FileCheck command line:  /home/chirag/repos/chirag-flang/build/bin/FileCheck /home/chirag/repos/chirag-flang/test/directives/vector_directive1.f90 --check-prefix=CHECK-NO-CLAUSE
```
I'll remove the `-S -emit-llvm` part.

Reply:
Done.